src/app/Http/Controllers/Auth/ForgotPasswordController.php

```diff
@@ -29,7 +30,9 @@ class ForgotPasswordController extends Controller
      */
     public function __construct()
     {
-        $this->middleware('guest');
+        $guard = config('backpack.base.guard') ? config('backpack.base.guard') : config('auth.defaults.guard');
```

Comment:
PHP 7 has the [null coalescing](http://php.net/manual/en/migration70.new-features.php#migration70.new-features.null-coalesce-op) operator `??`. `?:` works fine in php 5.3, it is also used in the Laravel source.